plugins/modules/network/routeros/routeros_api.py

```diff
@@ -0,0 +1,484 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Nikolay Dachev <nikolay@dachev.info>
+# GNU General Public License v3.0+ https://www.gnu.org/licenses/gpl-3.0.txt
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: routeros_api
+version_added: 1.1.0
+author: Nikolay Dachev (@nikolaydachev)
+short_description: Ansible module for RouterOS API
+description:
+  - Ansible module for RouterOS API with python librouteros.
+    This module can add, remove, update, query and execute arbitrary command
+    in routeros via api.
+options:
+  hostname:
+    description:
+      - RouterOS hostname api.
+    required: true
+    type: str
+  username:
+    description:
+      - RouterOS login user.
+    required: true
+    type: str
+  password:
+    description:
+      - RouterOS user password.
+    required: true
+    type: str
+  ssl:
+    description:
+      - If is set ssl will be used for RouterOS api connection.
+    required: false
+    type: bool
+  port:
+    description:
+      - RouterOS api port. If ssl is set, port will apply to ssl connection.
+        Defaults are http api - C(8728), https api - C(8729).
+    type: int
+  path:
+    description:
+      - Main path for all other arguments.
+        If other arguments are not set, api will return all items in selected path.
+        Example "ip address". Eqvivalent of RouterOS cli "/ip address print".
```

Comment:
```suggestion
        Example C(ip address). Eqvivalent of RouterOS cli C(/ip address print).
```
Same for the other code snippets below.

Reply:
nice suggestions! maybe `cli` should be capitalized as well?